run/CCCX.X-M-V1/RAMSIN.INCUS.V1

```diff
@@ -0,0 +1,855 @@
+  !     The SFC0 and UPA0 are the radii where the affect falls off to 0
+  !     Values are grid dependent.
+   
+   RODA_SFCE = 50000.,100.,100.,100.,
+   RODA_SFC0 = 100000.,100000.,100000.,100000.,
+   RODA_UPAE = 100000.,200.,200.,200.,
+   RODA_UPA0 = 200000.,2000.,2000.,2000.,
+   
+   RODA_HGT = 3000.,3000.,3000.,3000.,  ! Height at which transition from SFC radii 
+                                        !   to UPA radii occurs
+   
+   RODA_ZFAC = 100.,100.,100.,100.,     ! Vertical factor related to dx/dz
+   
+   ! - Time interpolate limit (TIL)- if the future-past obs time 
+   !    is > this limit, do not use to interpolate
+   !
+   ! - Time extrapolate limit (TEL)- if past/future obs is greater than TIL,
+   !    but less than TEL, use the obs
+   
+   ODA_SFC_TIL=21600.,
+   ODA_SFC_TEL=900.,
+   ODA_UPA_TIL=43200.,
+   ODA_UPA_TEL=21600.,
+!------------------------------------------------------------------------   
+
+! History start (if RUNTYPE='HISTORY')
+
+HFILIN = '/nobackup/pmarines/V/S/CA_B1_3G/T4/r/a-A-2018-12-13-183000-head.txt',
+                             ! Input state file name !CHANGEME
+
+! Analysis file input for assimilation: 1=yes, 0=no
+
+   IPAST_SFC = 0,            ! Initialize LEAF fields from history !CHANGEME for history restarts
+
+!------------------------------------------------------------------------
+   ICLOBBER = 1,             ! 0=stop if files exist, 1=overwite files
+!------------------------------------------------------------------------
+
+!  Analysis file output
+ 
+   IOUTPUT  =  1,             ! 0-no files, 1-write files
+   AFILEPREF  = '/nobackup/pmarines/V/S/CA_B1_3G/T4/G3/h1.1/a', !CHANGEME  
+                              ! File prefix for all analysis files
+                              !   state,mean,lite,both
+                              
+!------------------------------------------------------------------------
+   FRQSTATE   = 1800.,1800.,1800., ! "state" file write frequency !CHANGEME
+                              !  can be different for each grid
+                              !  works best if grids are multiples of each other
+   FRQST_KEEP   = 0.,0.,0.,      ! If > 0., frequency at which "state" files will be 
+                              !  kept. Others will be written, but deleted on
+                              !  the subsequent write. 
+                              !  Intended to be a mutiple of and >= FRQSTATE
+!-----------------------------------------------------------------------
+   FRQLITE  = 300,            ! Frequency for "lite" files !CHANGEME
+			      ! 300s (5 min) for G1&G2 / 30s for G3
+                              !   = 0 : no lite files
+   NLITE_VARS = 49,
+
+   LITE_VARS= 'THETA','RV','WP','VP','UP','PI','PP',                 ! 3D State (7)
+            'CCP','CDP','CRP','CHP','CGP','CPP','CAP','CSP',         ! 3D Number concentration (8)
+            'CCCNP','REGEN_AERO1_NP','CIFNP',                        ! 3D Aerosol # Concentrations (3)
```

Comment:
I've just noticed when looking through the mic_init file changes, that 'CCCNP' needs to be updated to 'CN1NP'! Otherwise we will be missing the CCN1 concentrations in the lite files

Reply:
Done!